qiita_db/analysis.py

```diff
@@ -30,6 +30,10 @@
 from qiita_core.exceptions import IncompetentQiitaDeveloperError
 from qiita_core.qiita_settings import qiita_config
 import qiita_db as qdb
+from qiita_db.archive import Archive
```

Comment:
Sorry one more, if we have `import qiita_db as qdb` we don't need `from qiita_db.archive import Archive`; simply do qdb.archive.Archive

Reply:
No problem.